bpf/aya-bpf/src/programs/sk_buff.rs

```diff
@@ -224,6 +225,37 @@ impl SkBuffContext {
             Err(ret)
         }
     }
+
+    /// Pulls data to the SKB linear section, returning
```

Comment:
This seems wrong? We return `()` on success.

I'd say:

```
Pulls in non-linear data in case the skb is non-linear.

Make len bytes from skb readable and writable. If a zero value is passed for `len`, then the whole length of the skb is pulled. This helper is only needed for reading and writing with direct packet access.
```

Reply:
Update as requested.